cmd/utils/nodecmd/consolecmd_test.go

```diff
@@ -53,15 +53,15 @@ func TestConsoleWelcome(t *testing.T) {
 	klay.SetTemplateFunc("gover", runtime.Version)
 	// TODO: Fix as in testAttachWelcome()
 	klay.SetTemplateFunc("klayver", func() string { return params.VersionWithCommit("") })
-	klay.SetTemplateFunc("niltime", func() string { return time.Unix(0, 0).Format(time.RFC1123) })
 	klay.SetTemplateFunc("apis", func() string { return ipcAPIs })
+	klay.SetTemplateFunc("datadir", func() string { return klay.Datadir })
 
 	// Verify the actual welcome message to the required template
 	klay.Expect(`
 Welcome to the Klaytn JavaScript console!
 
 instance: Klaytn/{{klayver}}/{{goos}}-{{goarch}}/{{gover}}
- datadir: {{.Datadir}}
+ datadir: {{datadir}}
```

Comment:
The original code is still working well. Why don't you use `.Datadir`?

Reply:
Just cosmetic reason, to make it consistent with `testAttachWelcome`.